src/lint/parser.ts

```diff
@@ -0,0 +1,31 @@
+export { Commit } from "@commitlint/parse";
+import parse, { Commit } from "@commitlint/parse";
+
+const blankCommit = {
+  raw: "",
+  header: "",
+  type: null,
+  scope: null,
+  subject: null,
+  body: null,
+  footer: null,
+  mentions: [],
+  notes: [],
+  references: [],
+  revert: null,
+  merge: null,
+};
+
+function createCommit(commit: Partial<Commit> = {}): Commit {
```

Comment:
I suppose this function is no longer need, `parseCommit` can clone `blankCommit` directly.